toolchain/driver/clang_runner_test.cpp

```diff
@@ -58,7 +58,8 @@ TEST(ClangRunnerTest, Version) {
   const auto install_paths =
       InstallPaths::MakeForBazelRunfiles(Testing::GetExePath());
   std::string target = llvm::sys::getDefaultTargetTriple();
-  ClangRunner runner(&install_paths, target, &test_os);
+  auto VFS = llvm::vfs::getRealFileSystem();
```

Comment:
Shouldn't this be "vfs", not "VFS", since it's a local variable?

Reply:
Sorry, I copied that from some llvm code. Fixed now.